community/modules/file-system/nfs-server/README.md

```diff
@@ -6,6 +6,10 @@ files with other clients over a network. `nfs-server` can be used by
 [vm-instance](../../../../modules/compute/vm-instance/README.md) and SchedMD
```

Comment:
I see the following behavior changes coming from this PR:
- We now may be installing python in addition to Ansible. Python takes much longer to install. People may be surprised to see a jump in startup time. I guess work around for now would be to tell people to use an older version of the script, or pick an image that already has python3.
- We are limiting access to the installed ansible behind the virtual environment. If I was installing ansible and expecting it to be available on my vm, I might be surprised to find it is not there.

Consider some documentation update.

Reply:
There is a problem here in that CentOS 7 is a massive outlier in that it does not have Python 3 baked in while everything else will. Of course, it's also very popular. The advice I gave @heyealex on this was not to use `yum install python3` but to use something like `yum install https://path/to/python3.rpm` where the URL is guaranteed (or nearly so) to live "forever".  We identified a few together. The advantage is that (I believe) it will bypass all repo analysis in favor of just trying to install the RPM (and dependencies). So it might need to be a short sequence of yum  (or `rpm -U`) install commands to install the dependencies in order. Likewise, we could consider a short-lived GCS/HTTPS mirror of these RPMs.

Reply:
I looked into using rpms for installing python, and the result was not very clean. It requires us to track down the dependencies ourselves, many of which also have dependencies. This doesn't seem like a good approach unfortunately.

For the yum install, I did add in the login to only use specified repos, just like what we did for the nfs-utils install. This should help cut down on installation time at least.